src/main/webapp/app/course/competencies/competency-management/import-all-competencies.component.ts

```diff
@@ -36,11 +37,15 @@ export type ImportAllFromCourseResult = {
 @Component({
     selector: 'jhi-import-all-competencies',
     templateUrl: './import-all-competencies.component.html',
+    standalone: true,
+    imports: [ArtemisSharedCommonModule, ArtemisSharedComponentModule],
 })
 export class ImportAllCompetenciesComponent extends ImportComponent<CourseForImportDTO> {
     //import relations by default
     protected importRelations = true;
 
+    @Input() public competencyType: 'competency' | 'prerequisite' | 'courseCompetency' = 'competency';
```

Comment:
what is meant by 'courseCompetency' = 'competency'? And could we not also use the CompetencyType enum here?

Reply:
Partially replaced with the CourseCompetencyType.
I use it in the html file to select the correct translation key. I'm open for other suggestions